sites/partners/src/components/listings/PaperListingForm/sections/LeasingAgent.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from "react"
 import { useFormContext } from "react-hook-form"
+import { isURL } from "class-validator"
```

Comment:
question: We have one other URL in the listing form called "custom online application URL". For that field we only rely on the backend validation. Can we do the same for this field as well?
![image](https://github.com/user-attachments/assets/c2be0ec0-e89a-4976-aba9-7ef7b7e6b91f)

Reply:
addressed in https://github.com/bloom-housing/bloom/pull/4807/commits/5b5370a15390c694c15231d8e5c329d36f9b6ad2